README.md

```diff
@@ -204,13 +204,35 @@ modules.
 sequencer.run();
 ```
 
+Sequencer can be run with a custom config object
+
+```js
+// The config object enables custom progress bars in node environment and
+// ability to run the sequencer from a particular index(of the steps array)
+
+sequencer.run(config);
+
+```
+
+The config object can have the following keys
+
+```js
+config: {
+  progressObj: , //A custom object to handle progress bar
+  index: //Index to run the sequencer from
+}
+```
+
 Additionally, an optional callback can be passed to this method.
 
 ```js
 sequencer.run(function(out){
```

Comment:
let's name the function `function callback(out)` to be extra clear. And on the line above, we can say `an optional callback function`. Great!!

Reply:
Oh great Idea!!